GUI/GPTMDTaskWindow.xaml

```diff
@@ -201,7 +276,12 @@
                         </StackPanel>
                     </Expander>
                 </GroupBox>
-                <GroupBox Header="G-PTM-D Modifications" DockPanel.Dock="Top">
+                <GroupBox Header="G-PTM-D Modifications" DockPanel.Dock="Top" ToolTipService.ShowDuration="999999" ToolTipService.InitialShowDelay="1000">
+                    <GroupBox.ToolTip>
+                        <TextBlock>
+                            Use for all mods that are possibly occuring in vivo (the more the merrier!)
```

Comment:
G-PTM-D uses matching unmodified fragments and a characteristic precursor mass shift to identify these

Reply:
i think we should remove "unmodified". If we're searching with xml database, then maybe some matched fragments have mods (or maybe oxidation or carbamidomethylation).

Reply:
"G-PTM-D uses uninformed fragment ions and characteristic precursor mass shifts to identify novel PTMs"

Reply:
"uninformed" is sort of jargon-y. They aren't really novel PTMs either, but novel PTM locations. How about:

G-PTM-D will search the database and use characteristic precursor mass-differences to identify these PTMs